src/torchvision_module.py

```diff
@@ -110,19 +118,81 @@ def get_attribute_from_config(attribute_name: str, default, of_type=None):
             normalize=(mean_rgb, std_rgb),
-            swap_R_and_B=swap_r_and_b,
+            swap_r_and_b=swap_r_and_b,
             channel_last=channel_last,
         )
 
         self.model.eval()
 
         self.labels_confidences = get_attribute_from_config(
-            "labels_confidences", dict()
+            "labels_confidences", {}
         )
         self.default_minimum_confidence = get_attribute_from_config(
             "default_minimum_confidence", 0
         )
+    #pylint: disable=too-many-arguments
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+        """Get the next image, detections, classifications, and objects all together,
+        given a camera name. Used for visualization.
+
+        ::
+
+            camera_name = "cam1"
+
+            # Grab the detector you configured on your machine
+            my_detector = VisionClient.from_robot(robot, "my_detector")
 
+            # capture all from the next image from the camera
+            result = await my_detector.capture_all_from_camera(
+                camera_name,
+                return_image=True,
+                return_detections=True,
+            )
+
+        Args:
+            camera_name (str): The name of the camera to use for detection
+            return_image (bool): 
+                Ask the vision service to return the camera's latest image
+            return_classifications (bool): 
+                Ask the vision service to return its latest classifications
+            return_detections (bool): 
+                Ask the vision service to return its latest detections
+            return_object_point_clouds (bool): 
+                Ask the vision service to return its latest 3D segmentations
+
+        Returns:
+            vision.CaptureAllResult: 
+                A class that stores all potential returns from the vision service.
+            It can  return the image from the camera along with its associated detections, 
+            classifications, and objects, as well as any extra info the model may provide.
+        """
+        result = CaptureAllResult()
+        image = await self.get_image_from_dependency(camera_name)
+
+        if return_image:
+            result.image = image
+        if return_classifications:
+            classifications = await self.get_classifications(image, 1)
+            result.classifications = classifications
+        if return_detections:
+            detections = await self.get_detections(image, timeout=timeout)
+            result.detections = detections
+        if return_object_point_clouds:
+            return NotImplementedError
```

Comment:
don't return NotImplementedError, rather just don't return anything (eliminate this if statement)